python/reg_interface/scripts/repeated_reg_read.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+from reg_utils.reg_interface.common.reg_xml_parser import getNode, parseXML
+from reg_utils.reg_interface.common.reg_base_ops import readAddress
+from reg_utils.reg_interface.common.reg_base_ops import writeReg
+from reg_utils.reg_interface.common.print_utils import printRed
+from reg_utils.reg_interface.common.jtag import initJtagRegAddrs
+from reg_utils.reg_interface.common.reg_base_ops import *
+
+#from rw_reg import *
+#from mcs import *
+
+import time
+import array
+import struct
+import os
+import sys
+import socket
+
+import argparse
+
+if __name__ == '__main__':
+
+    from optparse import OptionParser
```

Comment:
We have typically are more in favour of the `ArgParse` module for option parsing, see [Jared's comment](https://github.com/cms-gem-daq-project/reg_utils/pull/22#issuecomment-404862393) for some examples.